website/docusaurus.config.js

```diff
@@ -109,24 +113,6 @@ var siteSettings = {
           position: 'left',
           activeBasePath: 'reference'
         },
-        {
-          to: '/dbt-cli/cli-overview',
-          label: 'dbt CLI',
-          position: 'left',
-          activeBasePath: 'dbt-cli'
-        },
-        {
-          to: '/docs/dbt-cloud/cloud-overview',
-          label: 'dbt Cloud',
-          position: 'left',
-          activeBasePath: 'docs/dbt-cloud'
-        },
-        {
-          to: '/guides/getting-started',
-          label: 'Guides',
-          position: 'left',
-          activeBasePath: 'guides'
-        },
         {
           to: '/blog/',
           label: 'Developer Blog',
```

Comment:
"Developer blog" should also move under "Learn" (which currently says "Courses") per https://www.figma.com/file/FxfxSLnmL38WKHr1AR1LNT/DevHub-Top-Level-Navigation?node-id=12%3A393#258651098